### PR TITLE
Country profiles feedback - 1st round

### PR DESCRIPTION
**Pull request by andresgnlez**
Feedback based on [this document](https://docs.google.com/document/d/1930oZq1Hh0bozutdjvzZlenJ_jifNVGS/edit).

### CODE REVIEW

File: app/javascript/app/components/country/country-subnational-actions/country-subnational-actions-component.jsx [hunk 1]
```diff
@@ -110,11 +116,25 @@ function SubnationalActions({ indicators, loading, handleInfoClick }) {
     <div className={styles.noData}>No data available.</div>
   );
 
+  const countryName = useMemo(() => {
```
Comment (Bluesmile82) [inline app/javascript/app/components/country/country-subnational-actions/country-subnational-actions-component.jsx:122]: We normally do this on selectors but is ok 👌

**Review by Bluesmile82 — APPROVED** [review]
Great job! Thanks

**Comment by andresgnlez** [comment]
@Bluesmile82 I've tried, a couple of times, doing those selectors and, _for some _reason__, the props I was relying on sometimes are undefined leading to crashes, so I opted out for hooks, whenever possible. But yep, my initial intention was keeping the current structure with selectors.

**Comment by Bluesmile82** [comment]
No worries, for the future let's try with selectors. Maybe we can check that together to see what is happening

**Pull request merged**